README.md

```diff
@@ -12,21 +12,29 @@ Comes in two flavours, `:base` which is just PLV8 and PostGIS, and `:testing` wh
 To build the `base` image:
 
 ```sh
-docker build --target=base -t centreforea/postgres-plv8-postgis:plv8 .
+docker build --target=plv8 -t centreforea/postgres-plv8-postgis:plv8-14 .
```

Comment:
What's the reason for changing `base` to `plv8` explicitly? I can see it says `:base which is just PLV8 and PostGIS` but just curious

Reply:
If you only build the base then plv8 and posgis are missing